docs/source/datasets/index.rst

```diff
@@ -461,4 +461,148 @@ The following columns are included in these datasets:
+   * - Mailing address state
+     - :code:`mailing_address_state`  
+     - 
+   * - Mailing address ZIP code
+     - :code:`mailing_address_zipcode`
+     - 
+   * - Social security number 
+     - :code:`ssn`
+     - 
+   * - Individual tax identification number
+     - :code:`itin`
+     - 
+   * - Income 
+     - :code:`income`
+     - 
+   * - Employer ID 
+     - :code:`employer_id`
+     -  
+   * - Employer Name 
+     - :code:`employer_name`
+     - 
+   * - Employer street number
+     - :code:`employer_street_number`
+     - 
+   * - Employer street name
+     - :code:`employer_street_name`
+     - 
+   * - Employer unit number
+     - :code:`employer_unit_number`
+     - 
+   * - Employer city
+     - :code:`employer_city`    
+     - 
+   * - Employer state
+     - :code:`employer_state`  
+     - 
+   * - Employer ZIP code
+     - :code:`employer_zipcode`
+     - 
+   * - Joint filer first name
+     - :code:`spouse_first_name`
+     - Only included if the simulant has a joint filer (i.e., spouse).
+   * - Joint filer middle initial
+     - :code:`spouse_middle_initial`
+     - Only included if the simulant has a joint filer (i.e., spouse).
+   * - Joint filer last name
+     - :code:`spouse_last_name`
+     - Only included if the simulant has a joint filer (i.e., spouse).
+   * - Joint filer mailing address street number
+     - :code:`spouse_mailing_address_street_number`
+     - Only included if the simulant has a joint filer (i.e., spouse).
+   * - Joint filer mailing address street name
+     - :code:`spouse_mailing_address_street_name`
+     - Only included if the simulant has a joint filer (i.e., spouse).
+   * - Joint filer mailing address unit number
+     - :code:`spouse_mailing_address_unit_number`
+     - Only included if the simulant has a joint filer (i.e., spouse).
+   * - Joint filer mailing address PO box
+     - :code:`spouse_mailing_address_po_box`
+     - Only included if the simulant has a joint filer (i.e., spouse).
+   * - Joint filer social security number
+     - :code:`spouse_ssn`
+     - Only included if the simulant has a joint filer (i.e., spouse).
+   * - Joint filer individual tax identification number
+     - :code:`spouse_itin`
+     - Only included if the simulant has a joint filer (i.e., spouse).
+   * - Joint filer income
+     - :code:`spouse_income`
+     - Only included if the simulant has a joint filer (i.e., spouse).
+   * - Joint filer employer ID
+     - :code:`spouse_employer_id`
+     - Only included if the simulant has a joint filer (i.e., spouse).
+   * - Joint filer employer name
+     - :code:`spouse_employer_name`
+     - Only included if the simulant has a joint filer (i.e., spouse).
+   * - Joint filer employer address
+     - :code:`spouse_employer_address`
+     - Only included if the simulant has a joint filer (i.e., spouse).
+   * - Joint filer employer ZIP code
+     - :code:`spouse_employer_zipcode`
+     - Only included if the simulant has a joint filer (i.e., spouse).
```

Comment:
You say "spouse" and "joint filer" a lot... do we need to include both of these terms?

Reply:
I think the "only included if" wording is unclear. The DataFrame will still have these columns regardless of whether a simulant has a joint filer or not (same for dependents), but they will just be filled with NaN's in rows where they don't apply. I propose one of two things:

1. Just delete these notes. That would agree with what we've done for other values that may be missing, such as unit number.
2. Change the notes to something like "NaN if simulant does not file jointly" or "NaN if simulant has no dependents" or "NaN if simulant has fewer than 2 dependents," etc.